client/my-sites/plans/current-plan/jetpack-checklist/constants.js

```diff
@@ -58,3 +48,19 @@ export const JETPACK_CHECKLIST_TASKS = {
 		tourId: 'jetpackSignIn',
 	},
 };
+
+export const JETPACK_CHECKLIST_TASK_BACKUPS_REWIND = {
+	title: translate( 'Backups & Scanning' ),
```

Comment:
Should this be "Backup and Scan" as per p1HpG7-6z0-p2 ?

Reply:
+1 per our new naming convention!

Reply:
👍 Renamed.